modules/aws/auth.go

```diff
@@ -32,6 +33,21 @@ func NewAuthenticatedSession(region string) (*session.Session, error) {
 func NewAuthenticatedSessionFromDefaultCredentials(region string) (*session.Session, error) {
```

Comment:
Thanks for the PR! Could you please update the comment above this function to describe this behavior?

Reply:
Done.